custom-recipes/nlp-analysis-ontology-tagging/recipe.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import dataiku
+from dataiku.customrecipe import get_input_names_for_role, get_output_names_for_role
+from dku_plugin_config_loading import DkuConfigLoadingOntologyTagging, DkuConfigLoading
+from tagger import Tagger
+
+"""Return input dataframe and the list of its columns"""
+
+
+def get_input_dataframe(input_dataset):
+    input_name = get_input_names_for_role(input_dataset)[0]
+    dataframe = dataiku.Dataset(input_name).get_dataframe()
```

Comment:
I think the dataiku.Dataset part and the check for valid column names can be done by DkuConfigLoadingOntologyTagging directly